pkg/kubelet/volumemanager/populator/desired_state_of_world_populator.go

```diff
@@ -142,6 +142,10 @@ func (dswp *desiredStateOfWorldPopulator) Run(sourcesReady config.SourcesReady,
 		dswp.populatorLoop()
 		return done, nil
 	}, stopCh)
+
+	// Call dswp.populatorLoop here to process non PVC volumes first of all
+	dswp.populatorLoop()
```

Comment:
populatorLoop is already called in line 142?

Reply:
yes, it is duplicate.